protoc_plugin/CHANGELOG.md

```diff
@@ -3,9 +3,12 @@
 * Fix factory argument types for protobuf `Map` fields. ([#975])
 * Fix import order changes when files are passed in different order to `protoc`.
 ([#952])
+* Add fromDart() and toDart() methods to convert between core Duration and proto
```

Comment:
As a drive-by-comment, since this is adding new API, we should bump the minor version of the plugin (`22.0.2-wip` => `22.1.0-wip`). This will let people that know they need this API specify that in their dep (`^22.1.0`).